mycore-user2/src/main/resources/META-INF/resources/authorization/change-password.xed

```diff
@@ -5,7 +5,7 @@
     <xed:form class="userEditor form-horizontal" role="form" xmlns:xed="http://www.mycore.de/xeditor" xmlns:mcruser="http://www.mycore.de/components/mcruser">
       <xed:preload uri="xslStyle:user-xeditor-templates:webapp:authorization/user-editor.xed" />
       <xed:preload uri="xslStyle:user-xeditor-templates:webapp:{$MCR.UserForms.CustomIncludes}" />
-      <xed:source uri="user:{$id}" />
+      <xed:source uri="notnull:checkPermissionChain:administrate-users::user:{$id}" />
```

Comment:
should be `notnull:checkPermissionChain::administrate-users:user:{$id}` according to syntax